packages/components/src/components/SignIn.vue

```diff
@@ -1,62 +1,192 @@
 <template>
   <div class="signin-sidebar">
     <AppMapLogo width="120" />
-    <div class="content-wrap">
+    <div v-if="!submitted" class="content-wrap">
       <div class="sidebar-title">
-        <h1 data-cy="title">Sign in</h1>
-        <p>
-          Start visualizing your runtime code, identifying problems, and finding solutions. Sign in
-          via GitHub or GitLab to start automatically generating:
-        </p>
+        <h1 data-cy="title">Activate AppMap</h1>
       </div>
 
-      <ul class="signin-ctas">
-        <li>
-          <a href="https://appmap.io/docs/navie">AppMap Navie AI</a>
-        </li>
-        <li>
-          <a href="https://appmap.io/docs/diagrams">Code visualizations</a>
-        </li>
-        <li>
-          <a href="https://appmap.io/docs/analysis/in-ci.html">AppMap in CI</a>
-        </li>
-        <li>
-          <a href="https://appmap.io/docs/analysis/in-your-code-editor.html">Runtime analysis</a>
-        </li>
-      </ul>
       <div class="signin-buttons">
-        <a href="/" class="btn btn-primary" data-cy="sign-in-button" @click="signIn">Sign in</a>
+        <a
+          class="btn btn-primary"
+          data-cy="github-activation-button"
+          href="/"
+          @click.prevent="signIn"
+        >
+          Activate with <span><GitHubLogo class="small-logo" /> GitHub</span>
+        </a>
+        <a
+          class="btn btn-primary"
+          data-cy="gitlab-activation-button"
+          href="/"
+          @click.prevent="signIn"
+        >
+          Activate with <span><GitLabLogo class="small-logo" /> GitLab</span>
+        </a>
+        <div class="subscriber-notice">
+          <div class="pink-asterisk">*</div>
+          <p>You must activate with GitHub or GitLab if you are using an organizational license.</p>
```

Comment:
Why is this the case?

Reply:
I'm not sure, but Elizabeth mentioned it [here](https://docs.google.com/document/d/1k9RUstEYu1aDz1gFJgSe2Lav-Ls50BVOKu2DxpF1Sd4/edit?disco=AAABIeZDiUQ).

Reply:
This was removed